TrainingLoop/TrainingLoop.swift

```diff
@@ -18,7 +18,7 @@ import TensorFlow
 // Workaround https://bugs.swift.org/browse/TF-1122 that prevents us from registering a
 // loss function inside our TrainingLoop struct
 public final class LossFunctionWrapper<Output: Differentiable, Target> {
-  public typealias F = @differentiable (Output, @noDerivative Target) -> Tensor<Float>
+  public typealias F = @differentiable(Output, @noDerivative Target) -> Tensor<Float>
```

Comment:
```suggestion
  public typealias F = @differentiable (Output, @noDerivative Target) -> Tensor<Float>
```

I don't _think_ we'd normally drop that space.  Regardless, it's a good idea to avoid making needless whitespace changes on lines you don't otherwise have to edit.  If you want to do a whitespace cleanup commit, it should usually be a separate PR.